models/mpas_atm/model_mod.f90

```diff
@@ -7457,7 +7458,7 @@ subroutine compute_full_pressure(ens_size, theta, rho, qv, pressure, tk, istatus
 tk = theta_to_tk(ens_size, theta, rho, qv_nonzero, istatus)
 
 where (istatus == 0)       ! We only take non-missing tk here
-   pressure = rho * rgas * tk * (1.0_r8 + 1.61_r8 * qv_nonzero)
+   pressure = rho * rgas * tk * (1.0_r8 + rvord * qv_nonzero)
```

Comment:
i wanted to leave a comment on line 7445 - the 'dry density' comment should be 'dry air density' to be consistent.  i think that's the last place 'dry density' is in a comment (that i could find) that needs changing.  if all these suggested changes are ok, should we go ahead and accept them?

Reply:
@nancycollins I'm going to close this pull request and put the constant fix in cleanly. 
As Kevin says, it is probably best not to have the Jedi equations in the commits, and we need to add  documentation adding about the constant changes.

Reply:
@hkershaw-brown, just for clarification, there are no jedi equations or comments here at all. In this PR, I only updated the constants based on the model. I'm curious how you can make the fix cleaner?

Reply:
So the Jedi stuff is in the commit history:

![Screen Shot 2023-08-02 at 4 31 25 PM](https://github.com/NCAR/DART/assets/20047007/a049f7c3-ae6b-4f2a-b8cc-39ffa2b19f36)

Reply:
@syha I'm aware you're busy with other things, so I'll put the fix in (I'll add you as a joint commit so you show up in the contributions). Thanks, Helen

Reply:
Ah, the commit history, not the code itself. Got it. Thanks!